libspawner/src/win32/runner.cpp

```diff
@@ -136,6 +139,27 @@ void runner::restore_original_environment(const runner::env_vars_list_t& origina
     }
 }
 
+void runner::analyze_executable(const std::string& exe_path) {
+    LOADED_IMAGE exe_image;
+    std::unique_ptr<char> exe_path_rw(_strdup(exe_path.c_str()));
+    if (!MapAndLoad(exe_path_rw.get(), NULL, &exe_image, FALSE, TRUE)) {return;}
+
+    stack_exceeds_1gb = 0x40000000 < exe_image.FileHeader->OptionalHeader.SizeOfStackReserve;
```

Comment:
Ehm. I didn't understand the question.
It at least equal to the memory limit of the vast majority of problems, because default stack size on Windows is 1 MiB.